src/tribol/interface/tribol.cpp

```diff
@@ -86,6 +86,11 @@ void initialize( integer dimension, CommType comm )
    parameters.problem_comm = comm;
 
    internal::set_defaults( );
+
+   // set default logging level
+   axom::slic::finalize(); // TODO do we need to finalize here
```

Comment:
Does this call only happen once? or every cycle?

I don't think we want to finalize or initialize slic in general unless the user asks us to since slic is a singleton that is shared with other libraries.

Reply:
Correct. I removed the call to initialize and finalize. In general, these calls will be handled by the host-code. I am not going to introduce the ability for a user to initialize or finalize slic for now since this shouldn't be the general use case.